src/libraries/System.Text.Json/tests/System.Text.Json.Tests/Serialization/CustomConverterTests/CustomConverterTests.cs

```diff
@@ -180,10 +180,11 @@ void RunTest<TConverterReturn>()
         }
 
         [ActiveIssue("https://github.com/dotnet/runtime/issues/66232", TargetFrameworkMonikers.NetFramework)]
+        [SkipOnMono("https://github.com/dotnet/runtime/issues/66371")]
```

Comment:
If you want to just disable on mono interpreter (instead of all mono), you can use the `ActiveIssue` overload that takes a type and member names.

Similar to:
https://github.com/dotnet/runtime/blob/4e7501584427e72f62c72c07a2a1fafa6a3199e3/src/libraries/System.Text.Json/tests/System.Text.Json.Tests/Serialization/Stream.Collections.cs#L19